### ISSUE TITLE
Pretrained Sketch-a-net model

### ISSUE THREAD

**Issue by sawant-nidhish** [BasicCoder/SketchClassification#1]
Hello,
Thank you for your excellent work. Your PyTorch implementation of Sketch-a-net has helped me save a lot of effort. Can you please share a pre-trained model for Sketch-a-net? I want to fine-tune the network on a task which I am working on. It will be time-consuming and difficult for me to train the model from scratch. 

Please share a pre-trained model of sketch net and please mention the dataset on which it has been trained. I appreciate your efforts. Thank you once again.

**Comment by BasicCoder** [comment]
Thank you for your attention. I trained these models in the school laboratory, but now that I have graduated, I may have lost these model parameters. If I can find it this weekend, I will upload these model parameters. Perhaps the best solution is to retrain these models by yourself.

**Comment by sawant-nidhish** [comment]
@BasicCoder Actually, I tried retraining the model but I am not able to reproduce the results. I have used a 80/20 % split of TU-Berlin as Train/test. No changes were made to the code by my end. The loss always remains at 5.5 and preci@1 at 0.78
`Epoch: [50][105/125]	Time 1.481 (0.304)	Data 1.439 (0.249)	Loss 5.5218 (5.5216)	Prec@1 0.7812 (0.2948)	Prec@5 3.125 (1.607)`
`Epoch: [50][120/125]	Time 0.069 (0.284)	Data 0.000 (0.229)	Loss 5.5217 (5.5216)	Prec@1 0.0000 (0.2841)	Prec@5 0.000 (1.524)`

Please let me know what I should do to get the results right

**Comment by faridhasanov96** [comment]
Probably too late for you, but for future generations: 

I was also seeing no increase in score after training with ResNet 34; After careful inspection I found out that the difference was in data loading pipeline - I didn't shuffle the dataset before passing it to the loader. Somehow model wasn't able to learn on this data and was pretty much stuck at accuracy about 0.5% - but after applying the shuffling, I got results comparable with what is described in Readme.
